tests/helpers.py

```diff
@@ -609,6 +609,39 @@ def create_multiple_pvcs(
     ]
 
 
+def create_pvc_clone(
+        sc_name, parent_pvc_name, size=None, pvc_name=None, do_reload=True, volume_mode=None
+):
+    """
+    Create a cloned pvc from existing pvc
+    Args:
+        sc_name (str): The name of the storageclass
+        parent_pvc_name (str): The name of the pvc from which a cloned pvc would be created
+        pvc_name (str): The name of the PVC being created
+        namespace (str): The namespace for the PVC creation
```

Comment:
Can the namespace of cloned PVC be different from the namespace of parent PVC ?

Reply:
same not possible